p2p/libp2p/metrics/factory/connectionWatcherFactory.go

```diff
@@ -8,15 +8,18 @@ import (
 	"github.com/ElrondNetwork/elrond-go/p2p/libp2p/metrics"
 )
 
-const printConnectionsWatcher = "print"
-const disabledConnectionsWatcher = "disabled"
+const (
+	printConnectionsWatcher    = "print"
```

Comment:
move them in a new package and export them & use everywhere

Reply:
why exported?

Reply:
added them to p2p and used everywhere